testsuite/tests/lib-list/test.ml

```diff
@@ -11,6 +11,9 @@ let string_of_even_opt x =
 let () =
   let l = List.init 10 (fun x -> x) in
   let sl = List.init 10 string_of_int in
+  assert (List.iota 3 = [0; 1; 2]);
+  assert (List.iota 5 ~start:1 = [1; 2; 3; 4; 5]);
+  assert (List.iota 5 ~start:10 ~step:2 = [10; 12; 14; 16; 18]);
```

Comment:
Shouldn't the length be after the optional arguments?

Reply:
I used the label-bearing functions in the `Option` module as a reference, where e.g. `Option.fold` also defines the labelled arguments before the unlabelled ones.

Reply:
Oh, sorry, I see that you're talking about the tests. Yes, that should be changed for clarity. I did sent new commits with changed documentation and changed tests.